signup.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>login</title>
+</head>
+<body>
+    <div class="logo_box">
+        <img class="logo" src="images/login/logo.png">
```

Comment:
❗️ 수정요청
`"판다마켓" 클릭 시 루트 페이지("/")로 이동합니다.(새로고침)` 를 구현하기 위해 해당 이미지 a 태그로 감싸주세요~
index.html의 header 컴포넌트의 로고를 생각하시면 좋을 것 같아요~

```suggestion
        <a href="/"><img class="logo" src="images/login/logo.png" alt="판다마켓"></a>
```